core/src/main/java/hudson/util/Iterators.java

```diff
@@ -403,4 +405,20 @@ public void remove() {
     public interface CountingPredicate<T> {
         boolean apply(int index, T input);
     }
+
+    /**
+     * Similar to {@link com.google.common.collect.Iterators#skip} except not {@link Beta}.
+     * @param iterator some iterator
+     * @param count a nonnegative count
+     * @since FIXME
+     */
+    public static void skip(@Nonnull Iterator<?> iterator, int count) {
+        if (count < 0) {
+            throw new IllegalArgumentException();
+        }
+        while (iterator.hasNext() && count-- > 0) {
+            iterator.next();
```

Comment:
A bit simpler than the Guava version; we do not care about returning the skip count.